composer.json

```diff
@@ -10,6 +10,18 @@
     "drupal": {
       "type": "composer",
       "url": "https://packages.drupal.org/8"
+    },
+    "ckeditor5-anchor-drupal": {
```

Comment:
@nJim This question for after Thanksgiving 😄 And also to add some background incase someone else ends up picking this up. 

The anchor link module relies on this package to work properly, but I was getting errors adding it to the profile composer.json file. However adding it here allowed it to install without issue. Is it alright to leave this here, or would it be better to find a way move it to the profile composer.json file?

Reply:
Hi @hectorlj! Yup, since the profile `composer.json` is the correct place to add the `ckeditor-anchor-drupal` repo, you also have to add that `ckeditor-anchor-drupal` to the `repositories` section in the profile `composer.json` to get that one to load correctly. This is for security reasons to make sure you aren't loading other repos you didn't intend.

Reply:
Ah, gotcha. Thanks @codechefmarc!

Reply:
One last thing, then you've got it - please move the `"northernco/ckeditor5-anchor-drupal": "^0.3.0",` from the root `composer.json` to the profile `composer.json`. Thank you!

Reply:
@codechefmarc Just double checked the composer docs and it looks like the packages in the `repositories` section is only read from the root `composer.json`... I'll double check to see if there's another way to get that package installed
https://getcomposer.org/doc/04-schema.md#repositories

Reply:
Thanks for looking into it, @hectorlj! Let me know what you find. It's been a while since I did something like that so I forget exactly the placement, but I wonder if it needs to be in both places? I think I tested that and it worked? I hate to have redundant code though.

Reply:
@codechefmarc Looks like it works when it's in both places 🤷🏼‍♂️ I did try using `wikimedia/composer-merge-plugin` to use the composer file in the anchor link module, but it had its own issues with the root composer file